api-abandoned-workloads.md

```diff
@@ -0,0 +1,13 @@
+# Abandoned Workloads
+
+The abandoned workloads API suggests cluster workloads that have been abandoned based on network traffic levels. It is available at:
+```
+http://<kubecost-address>/model/savings/abandonedWorkloads
+```
+
+## Parameters
+
+| Name | Type | Description |
+|------|------|-------------|
+| `days` | int | Number of historical days over which network traffic should be measured. |
+| `threshold` | int | The threshold of traffic at which a workload is determined abandoned. |
```

Comment:
@mbolt35 Do you know what the units are here?

Reply:
bytes/second

Reply:
Thanks!